docs/ros-bridge.md

```diff
@@ -0,0 +1,67 @@
+
+- `/fsds_ros_bridge/VEHICLE_NAME/global_gps` [sensor_msgs/NavSatFix](https://docs.ros.org/api/sensor_msgs/html/msg/NavSatFix.html)   
+This the current GPS coordinates of the drone in airsim. 
+
+- `/fsds_ros_bridge/VEHICLE_NAME/odom_local_ned` [nav_msgs/Odometry](https://docs.ros.org/api/nav_msgs/html/msg/Odometry.html)   
+Odometry in NED frame wrt starting point.  THIS WILL NOT BE STREAMED DURING COMPETITION.
+
+- `/fsds_ros_bridge/VEHICLE_NAME/CAMERA_NAME/IMAGE_TYPE/camera_info` [sensor_msgs/CameraInfo](https://docs.ros.org/api/sensor_msgs/html/msg/CameraInfo.html)
+
+- `/fsds_ros_bridge/VEHICLE_NAME/CAMERA_NAME/IMAGE_TYPE` [sensor_msgs/Image](https://docs.ros.org/api/sensor_msgs/html/msg/Image.html)   
+  RGB or float image depending on image type requested in [settings.json](../UE4Project/Plugins/AirSim/Settings/settings.json).
+
+- `/tf` [tf2_msgs/TFMessage](https://docs.ros.org/api/tf2_msgs/html/msg/TFMessage.html)
+
+where `VEHICLE_NAME`, `CAMERA_NAME` and `IMAGE_TYPE` are extracted from [settings.json](../UE4Project/Plugins/AirSim/Settings/settings.json).
+
+## Subscribers
+- `/fsds_ros_bridge/VEHICLE_NAME/control_command` [fsds_ros_bridge/ControlCommand](../Simulator/src/fsds_ros_bridge/msg/ControlCommand.msg) 
+The contents of this message fill the essential parts of the `msr::airlib::CarApiBase::CarControl` struct. This is the only way to control the car when the airsim ROS client is connected (keyboard will no longer work!).
+
+## Services
+
+- `/fsds_ros_bridge/reset` [fsds_ros_bridge/Reset](../Simulator/src/fsds_ros_bridge/srv/Empty.html)
+ Resets car to start location.
+
+## Parameters
+- `/fsds_ros_bridge/update_airsim_control_every_n_sec` [double]   
+  Set in: `$(fsds_ros_bridge)/launch/fsds_ros_bridge.launch`   
+  Default: 0.01 seconds.    
+  Timer callback frequency for updating drone odom and state from airsim, and sending in control commands.    
+  The current RPClib interface to unreal engine maxes out at 50 Hz.   
+  Timer callbacks in ROS run at maximum rate possible, so it's best to not touch this parameter. 
+
+- `/fsds_ros_bridge/update_airsim_img_response_every_n_sec` [double]   
+  Set in: `$(fsds_ros_bridge)/launch/fsds_ros_bridge.launch`   
+  Default: 0.01 seconds.    
+  Timer callback frequency for receiving images from all cameras in airsim.    
+  The speed will depend on number of images requested and their resolution.   
+  Timer callbacks in ROS run at maximum rate possible, so it's best to not touch this parameter. 
+
+## Visualization
+This package contains some useful launch and config files which will help you in visualizing the data being streamed through the above topics.
+
+To open Rviz with [this](../Simulator/src/fsds_ros_bridge/config/rviz/default.rviz) configuration file, run `roslaunch fsds_ros_bridge rviz.launch`.
+
+To open Multiplot with [this](../Simulator/src/fsds_ros_bridge/config/multiplot/multiplot.xml) configuration file, run `roslaunch fsds_ros_bridge plot.launch`
```

Comment:
Simulator directory no longer exists. Do a find 'Simulator' and replace with 'ros'